src/test/scala/com/gd/twitteranalytics/util/ApplicationConfigReaderTest.scala

```diff
@@ -0,0 +1,28 @@
+package com.gd.twitteranalytics.util
+
+import com.typesafe.config.ConfigException
+import org.scalatest.FreeSpec
+
+class ApplicationConfigReaderTest extends FreeSpec {
+
+  "Methods tests" - {
+    "getAppConfigurables" - {
+      "should throw ConfigException error if Application configurations are missing" in {
```

Comment:
> throw ConfigException error

the statement is a bit misleading, site the exception is not an error.
dropping "error" from the statement would make things clearer.

Reply:
Updated test case